### PR TITLE
Feature/event history

### PR DESCRIPTION
**Pull request by NombusoRR**
feture/EventHistory branch merge to dev branch

### CODE REVIEW

File: waterlog/src/Globals.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
 export const Globals = {
-	API_URL: 'http://api.iot.retrotest.co.za'
+    API_URL: 'https://localhost:44382'
```
Comment (HeinOldewageRetro) [inline waterlog/src/Globals.js]: I don't think this is what you want. I understand having to change it on the local machine for testing. Maybe consider using environment variables, that way the code is the same but can be changed depending on where it is built. 

https://medium.com/@trekinbami/using-environment-variables-in-react-6b0a99d83cf5

File: waterlog/src/Pages/Map.js [hunk 2]
```diff
@@ -1,10 +1,11 @@
 import React, {Component} from 'react';
+import HistoryComponent from './History';
 
 class MapComponent extends Component {
   render() {
     return ( 
     <div className = "App" >
-      <div> Map Component </div> 
+      <div> <HistoryComponent id={2}/> </div>
```
Comment (CeeyarRbt) [inline waterlog/src/Pages/Map.js:8]: why 2?

File: waterlog/src/Components/HistoryTable.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+			return HighStatusIcon();
+		case 'low':
+			return LowStatusIcon();
+		case 'medium':
+			return MediumStatusIcon();
+		default:
+			return null;
+	}
+};
+class HistoryTableComponent extends Component {
```
Comment (laurent-mbuyu) [inline waterlog/src/Components/HistoryTable.js:33]: Your component makes use of props.
Have a constructor that receives props as a parameter and make a call to the super function having a props parameter too.
```
constructor(props)
{
    super(props)
}
```

File: waterlog/src/Pages/Map.js (hunk 2 above)
Comment (NombusoRR) [reply waterlog/src/Pages/Map.js]: Are you asking why the component History Component is nested inside the Map Component, or about the hard coded ID?

File: waterlog/src/reducers/index.js [hunk 4]
```diff
@@ -2,15 +2,21 @@ import { combineReducers } from 'redux';
...
 import LeakLitresReducer from './LeakLitresReducer';
+import SegmentLeaksHistoryReducer from './SegmentLeaksHistoryReducer';
 import LeaksResolvedReducer from './LeaksResolvedReducer';
 import SegmentEventsReducer from './SegmentEventsReducer';
 
 export default combineReducers({
 	leaks: SegmentLeaksReducer,
 	leak: SegmentLeaksByIdReducer,
 	litres: LeakLitresReducer,
+<<<<<<< HEAD
```
Comment (laurent-mbuyu) [inline waterlog/src/reducers/index.js]: You have merge conflicts. Attend them before pushing your code.

File: waterlog/src/Pages/Map.js (hunk 2 above)
Comment (NombusoRR) [reply waterlog/src/Pages/Map.js:8]: Never mind the first comment, I had missed the 2 in your question.

For testing purposes, once the page is linked to a page which belongs to a pending feature, will use props to pass it the id

Comment (CeeyarRbt) [reply waterlog/src/Pages/Map.js:8]: Okay, but I think you should avoid pushing static test data `might cause unnecessary issues` well this was just out of curiosity so ...as you were.

File: waterlog/src/Globals.js (hunk 1 above)
Comment (NombusoRR) [reply waterlog/src/Globals.js]: I have tried environment variables, couldn't get it to work, will keep working on them.

File: waterlog/src/Components/HistoryTable.js (hunk 3 above)
Comment (NombusoRR) [reply waterlog/src/Components/HistoryTable.js:33]: Done

File: waterlog/src/reducers/index.js (hunk 4 above)
Comment (NombusoRR) [reply waterlog/src/reducers/index.js]: Done

**Comment by NombusoRR** [comment]
> sweet be aware of merge conflicts be you push , always make sure you pull the lastest remote Dev brance(or the branch you want to create a pull request for before your changes

Thank you, noted

**Review by CeeyarRbt — APPROVED** [review]

**Review by ash-rabbit — APPROVED** [review]

**Pull request merged**